docs/source/devel/index.rst

```diff
@@ -50,19 +50,20 @@ The repository Worker has two maintenance tasks:
 
 - **Bump Roles** that contain online keys ("Snapshot", "Timestamp" and Hahsed
   Bins ("bins-XX").
-- **Publish the new Hashed Bins Roles** ("bins-XX") with new/removed targets.
+- **Publish the new Hashed Bins Target Roles** ("bins-XX") with new/removed
+  targets.
 
 About **Bump Roles** (``bump_online_roles``) that contain online keys is easy.
 These roles have short expiration (defined during repository configuration) and
 must be "bumped" frequently. The implementation in the RepositoryMetadata
 
-**Publish the new Hashed Bins Roles** (``publish_targets_meta``) is part of the
+**Publish the new Hashed Bins Target Roles** (``publish_targets``) is part of the
 solution for the :ref:`Repository Worker scalability, Issue 17
 <devel/known_issues:(Solved) Scalability>`.
 
 To understand more, every time the API sends a task to add a new target, the
-Hashed Bins Roles must be changed to add the new target(s), as the Snapshot and
-Timestamp.
+Hashed Bins Roles must be changed to add the new target(s), followed by a new
+Snapshot and Timestamp versions.
```

Comment:
Even though timestamp, snapshot and hashed bins are important for us I don't see why they are with upper case letters.
I have the same opinions for `Repository Worker` and `Repository Metadata backend`.

What do you think?

Reply:
I use it to highlight the role name (not to confuse it with the generic word snapshot timestamp) and the component name (Repository Worker).
We do it across many docs. My suggestion is if we want to normalize it do a specific issue and change across all doc.

Reply:
Okay, let's leave it like that for now.